cookbooks/aws-parallelcluster-platform/resources/fabric_manager/partial/_fabric_manager_install_rhel.rb

```diff
@@ -14,9 +14,12 @@
 
 action :install_package do
   package 'yum-plugin-versionlock'
-
-  package fabric_manager_package do
-    version fabric_manager_version
-    action %i(install lock)
+  bash "Install #{fabric_manager_package}" do
```

Comment:
What about adding retry for the installation like what we do for debian?
For example, in debian we have:
```
action :install_package do
  # For ubuntu, CINC17 apt-package resources need full versions for `version`
  execute "install_fabricmanager_for_ubuntu" do
    command "apt -y install #{fabric_manager_package}=#{fabric_manager_version} "\
            "&& apt-mark hold #{fabric_manager_package}"
    retries 3
    retry_delay 5
  end
end
```